docs/FORMATS_GUIDE.md

```diff
@@ -14,13 +14,22 @@ terraform-docs [PATH] [flags]
 
 ```
   -c, --config string               config file name (default ".terraform-docs.yml")
+      --footer-from string          relative path of a file to read footer from (default "")
       --header-from string          relative path of a file to read header from (default "main.tf")
   -h, --help                        help for terraform-docs
+<<<<<<< HEAD
```

Comment:
It seems there was a problem while fixing the conflict for all the `docs/formats` files. You can just run `make docs` to regenerate instead of attempting to fix and clean up these.

Reply:
Sorry I probably should have been clearer.
Cause of the changes required in plugin-sdk `make docs` doesn't work.
I just pushed up unchanged knowing they'd be regenerated anyway

Reply:
Oh I see, sounds good. Since we've merged your PR on plugin-sdk, you can go ahead and update `go.mod` accordingly.